node/package_json.ts

```diff
@@ -1,8 +1,22 @@
-import { createRequire } from 'module'
+import { readFileSync } from 'fs'
 
-const require = createRequire(import.meta.url)
-const pkgJson = require('../package.json')
+import { findUpSync } from 'find-up'
 
-const getPackageVersion = (): string => pkgJson.version
+const getPackageVersion = () => {
```

Comment:
This could use a refactor to make it async and return something better than an empty string when something goes wrong, but that involves changing Netlify Build and I'd like to avoid doing that today if possible. I'll revisit this some other time.